### PR TITLE
[4.0] Rename presets for main menu

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #26067

### Summary of Changes
The names of presets for main menu have been changed

### Testing Instructions
In Modules, open the admini module "admin menu" and change the preset. 

### Expected result
Everything works as before

### CODE REVIEW

**Comment by infograf768** [comment]
I guess we have to add in script the deletion of the older joomla and modern presets.
Looks like changing file names is not really understood by git which consider it is a conflict.

File: administrator/modules/mod_menu/Menu/CssMenu.php [hunk 1]
```diff
@@ -132,7 +132,7 @@ public function load($params, $enabled)
 				$heading = new MenuItem(['title' => 'MOD_MENU_RECOVERY_MENU_ROOT', 'type' => 'heading']);
 				$this->root->addChild($heading);
 
-				MenusHelper::loadPreset('joomla', true, $heading);
+				MenusHelper::loadPreset('defaulta', true, $heading);
```
Comment (infograf768) [inline administrator/modules/mod_menu/Menu/CssMenu.php]: Typo. An "a" too much here. Should be `default`

Comment (chmst) [reply administrator/modules/mod_menu/Menu/CssMenu.php]: oops, thx

**Comment by infograf768** [comment]
I test this OK btw if the so-called conflicts can be solved.

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 733d8232412ad012d68e7c18af5d51df09a9ceed<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26084">issues.joomla.org/tracker/joomla-cms/26084</a>.</sub>

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 733d8232412ad012d68e7c18af5d51df09a9ceed<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26084">issues.joomla.org/tracker/joomla-cms/26084</a>.</sub>

**Comment by infograf768** [comment]
RTC if conflicts can be solved

@zero-24 @HLeithner @wilsonge <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26084">issues.joomla.org/tracker/joomla-cms/26084</a>.</sub>

**Comment by zero-24** [comment]
Yes already on my list.

**Comment by zero-24** [comment]
merge conflict has been resolved

**Comment by infograf768** [comment]
Restarted drone

**Comment by infograf768** [comment]
@wilsonge @HLeithner 
Please merge so that we can improve the alternate xml

**Comment by infograf768** [comment]
Thanks!

**Pull request merged**